### ISSUE TITLE
Apply TensorFlow on ARFF

### ISSUE THREAD

**Issue by dinasaif** [tensorflow/tensorflow#6639]
I have ARFF file or excel sheet with binary features(0,1) and class label also (0,1) can you help me to apply convolution neural network on my file to make classification with TensorFlow .
Thanks

**Comment by vrv** [comment]
NOTE: Only file GitHub issues for bugs and feature requests.  All other topics will be closed.

For general support from the community, see [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow).
To make bugs and feature requests more easy to find and organize, we close issues that are deemed
out of scope for GitHub Issues and point people to StackOverflow.
